test/testsimplifytypedef.cpp

```diff
@@ -2508,6 +2509,22 @@ class TestSimplifyTypedef : public TestFixture {
         ASSERT_EQUALS("", errout.str());
     }
 
+    void simplifyTypedef124() { // ticket #7792
```

Comment:
I assume this simplifyTypedef stuff is here by mistake

Reply:
yes, this commit for pull request #1334 . It is mistake when create request